Tech Lead/Minutes/20250204.md

```diff
@@ -0,0 +1,53 @@
+# Technical Lead Development Team call 4 February 2025
+
+## Participants
+
+- Attendees: @bettina-gier @schlunma @bouweandela @valeriupredoi @charles-turner-1 @rbeucher @sloosvel
+- Absentees: @ehogan @flicj191
+- Chair: @valeriupredoi
+- Minutes taker: @valeriupredoi
+  - Names legend:
+    - Bouwe Andela [@bouweandela](https://github.com/bouweandela)
+    - Saskia Sloosveldt [@sloosvel](https://github.com/sloosvel)
+    - Valeriu Predoi [@valeriupredoi](https://github.com/valeriupredoi)
+    - Manuel Schlund [@schlunma](https://github.com/schlunma)
+    - Emma Hogan [ehogan](https://github.com/ehogan)
+    - Bettina (Tina) Gier [@bettina-gier](https://github.com/bettina-gier)
+    - Romain Beucher [@rbeucher](https://github.com/rbeucher)
+    - Felicity Chun [@flicj191](https://github.com/flicj191)
+    - Charles Turner [@charles-turner-1](https://github.com/charles-turner-1)
+
+## Action Items (from 7 January: two done, one new one pending)
+
+| Meeting | Assignees | Item | Status |
+|-|-|-|-|
+| 2024-12-03 | @rbeucher | Arrange meeting on Intake | happened |
+| 2024-12-03 | ~~@ehogan~~ @bouweandela | Split docs meeting | happened, @flicj191 will start working on this, see https://github.com/esmValGroup/esMValTool/issues/3860 for further information |
+| 2025-02-04 | @valeriupredoi | set up TLT-SLT call for broken recipes (see below) | Pending |
+
+## Agenda and topics discussed
+
+- Pending actions: both have been addressed: docs split will be started by @flicj91 (cheers!) and meeting about the Intake lib has happened
+- @schlunma: broken recipes discussed in detail:
+SLT discussed [broken recipe policy](https://docs.esmvaltool.org/en/latest/community/broken_recipe_policy.html)
+    - Current policy:
+      - Broken recipe and diagnostic code are removed from the ESMValTool main branch
+      -> @bouweandela raises the issue of complete removal (NOT for an example like Julia recipe!), there are ways of "archiving" old code but not removing, but those are difficult on the technical side
+      - Scientific documentation of the recipe (and diagnostics) will be kept in the user and developer guide with an additional note and link to the last release in which the recipe was still fully functional
+      -> @bouweandela: code will not actually work since deps are too old, unlikely to have an out-the-box solution - an idea to have them on Zenodo, @valeriupredoi: since both deps and data will become unavailable (could bundle conda-lock files) it'll be difficult
```

Comment:
```suggestion
      -> @bouweandela: code will not actually work since deps are too old, unlikely to have an out-the-box solution - current idea is to have them on Zenodo for archival/provenance and as Docker Containers for reproducibility, git tags are available for convenient access @valeriupredoi: since both deps and data will become unavailable (could bundle conda-lock files) it'll be difficult
```